precode.sh

```diff
@@ -1,43 +1,63 @@
 #!/bin/bash
 
 # создаём каталог task с вложенными директориями
+mkdir -p ./task/dir1 ./task/dir2 ./task/dir3/dir4
 # task
 #   dir1
 #   dir2
 #   dir3
 #       dir4
 
 # изменяем текущую директорию на task
+cd ./task
```

Comment:
`./` можно убрать во всех командах, кроме 46 строчки
```suggestion
cd task
```